packages/crayons-core/src/components/country-phone/country-phone.tsx

```diff
@@ -0,0 +1,357 @@
+
+  /**
+   * Inner hint text displayed below the text box.
+   */
+  @State() innerHintText = '';
+
+  /**
+   * Country phone code
+   */
+  @State() phoneCode: string;
+
+  /**
+   * Name of the component, saved as part of form data.
+   */
+  @Prop() name = '';
+  /**
+   * Displays a right-justified clear icon in the text box. Clicking the icon clears the input text. If the attribute’s value is undefined, the value is set to false. For a read-only input box, the clear icon is not displayed unless a default value is specified for the input box.
+   */
+  @Prop() clearInput = false;
+
+  /**
+   * Adjust country code select dropdown width
+   */
+  @Prop() countryWidth?: string | null = '200';
+  /**
+   * Hide Country Name in the Country Select
+   */
+  @Prop() hideCountryName = false;
+  /**
+   * Hide Country Flag in the Country Select
+   */
+  @Prop() hideCountryFlag = false;
+
+  /**
+   * Text displayed in the text box before a user enters a value.
+   */
+  @Prop() selectPlaceholder?: string | null;
+
+  /**
+   * Text displayed in the text box before a user enters a value.
+   */
+  @Prop() inputPlaceholder?: string | null;
+
+  /**
+   *  * Label displayed on the interface, for the input component.
+   */
+  @Prop() inputLabel?: string | null;
+
+  /**
+   *  * Label displayed on the interface, for the input component.
+   */
+  @Prop() selectLabel?: string | null;
+
+  /**
+   * If true, the user cannot enter a value in the input box. If the attribute’s value is undefined, the value is set to false.
+   */
+  @Prop() readonly = false;
+
+  /**
+   * Specifies the input box as a mandatory field and displays an asterisk next to the label. If the attribute’s value is undefined, the value is set to false.
+   */
+  @Prop() required = false;
+
+  /**
+   * Specifies hint from inside the component after valdating phone number.
+   */
+  @Prop() requiredInnerHint = false;
+
+  /**
+   * Disables the component on the interface. If the attribute’s value is undefined, the value is set to false.
+   */
+  @Prop() disabled = false;
+
+  /**
+   * Hint text displayed below the text box.
+   */
+  @Prop() hintText = '';
+  /**
+   * Warning text displayed below the text box.
+   */
+  @Prop() warningText = '';
+  /**
+   * Error text displayed below the text box.
+   */
+  @Prop() errorText = '';
+
+  /**
+   *
+   * Theme based on which the text box is styled.
+   */
+  @Prop() state: 'normal' | 'warning' | 'error' = 'normal';
+
+  /**
+   * Default value displayed in the input box & select dropdown after extracting valid phone number
+   */
+  @Prop({ mutable: true }) value?: string | null = '';
+
+  // Events
+  /**
+   * Triggered when phone element is input.
+   */
+  @Event() fwTelInput: EventEmitter;
+
+  /**
+   * Triggered when phone element is blur.
+   */
+  @Event() fwTelBlur: EventEmitter;
+
+  /**
+   * Triggered when clear icon is clicked.
+   */
+  @Event() fwTelInputClear: EventEmitter;
+
+  @Method()
+  async getLibMethods() {
```

Comment:
Since we have just two library methods, kindly consider exposing them directly from the component. 
fwCountryPhone.isValidPhoneNumber, fwCountryPhone.parsePhoneNumber.

Reply:
Fixed ✅